### ISSUE TITLE
Sort by module name, repository owner & GitHub stars

### ISSUE THREAD

**Issue by bennycode** [mouafa/module-finder#1]
Very nice project! It would be cool if list items can be sorted by name, owner & stars.

**Comment by mouafa** [comment]
Hi @bennyn,
I think so, but unfortunately the API of libraries.io which the app consume doesn't support sorting.
I'm thinking of building a custom API service for module searching, but this might take sometime.

**Comment by bennycode** [comment]
But if you receive the search result you could sort it with JavaScript on the client-side, no?

**Comment by mouafa** [comment]
Right, but the sort will be just for the current displayed data context, so if the user sort the first result page with stars, then he move to the second page, he'll notice that the data is not sorted in a proper way, because he think that we sorted all the result for him, but we just sorted the current displayed results, and that's in my opinion a fail in term of usability and UX.
